packages/fe-fpm-writer/test/unit/prompts/__snapshots__/prompts.test.ts.snap

```diff
@@ -1211,6 +1230,24 @@ exports[`Prompts getCodeSnippet Type "filter-bar", get code snippet, min ui5Vers
 />"
 `;
 
+exports[`Prompts getCodeSnippet Type "filter-bar", get code snippet, min ui5Version = 1.96.25 1`] = `
+"<macros:FilterBar
+    id=\\"id\\"
+    metaPath=\\"/entity/qualifier\\"
```

Comment:
@AlinaGovoruhina  I think it is bug, on version `1.96.25` it should use `contextPath`. @hitesh-parmar  please correct if I am wrong

Reply:
Yes, this is correct. For all version 1.96.x there should be a contextPath

Reply:
This is fixed.